### DISCUSSION TITLE
How PhotoPrism recongnize that I am free or paid user?

### DISCUSSION THREAD

**Discussion by npodbielski** [photoprism/photoprism#3064] [Q&A] [answered]
I looked through docs, your patreon page, github sponsors, Photo Prism settings. 
Nowhere I did find anything about setting up paid account or including license key or license file or anything like that. 
How does PhotoPrism will be able to tell if I actually sponsor your project or not?

Also checked if there is separate PhotoPrism docker image for paying users. There is not. 

How does this work exactly?

**Comment by lastzero** [comment, marked as answer]
We cannot tell you this in our community forum. However, the source code is public for you to review and it's easy to sign up! ❤️

**Reply by npodbielski** [reply]
Haha, yeah I got you! :d
I do not want to hack PP. I am pretty sure I could - too lazy for that though :)
I just want to know what steps I do need to take to make PP recognize that I actually paid. As a dumb user. :)

But if you do not want then I we can close this topic.

**Reply by lastzero** [reply]
Are you a sponsor? We can tell you in private if you contact us by email :)

**Reply by npodbielski** [reply]
I am planning to became one. I just like to know things in advance instead of jumping into hot water - if you know what I mean :)

Ok, you we close this thread I guess. Thanks for help!

**Reply by lastzero** [reply]
I'd say your risk is very limited :+1:

**Reply by drealph90** [reply]
You just need to quit being such a holes and tell people how how to tell photoprism they subscribed publicly so they'll stop asking you and taking up your time with private emails. In other words quit being greedy and going against the spirit of Open source software, it's bad enough that open source software has a subscription model to begin with. If anything you should charge a one-time fee instead of monthly

**Reply by npodbielski** [reply]
Chill out dude. This is their own software. It is they decision how they monetize it if at all. 
If you do not like it do not use it. Write your own, or fork this one. 
This is true spirit of OSS :)
